src/v1/services/userService.ts

```diff
@@ -88,4 +88,16 @@ export default class UserService {
       throw error;
     }
   }
+
+  static async getUserById(userId: number): Promise<User> {
+    try {
+      const user = await db.getUserById(userId);
+      if (typeof user === "string") {
+        throw Error(user);
+      }
```

Comment:
you throw error in the user.ts, so is this even necessary? Don't you just catch and throw it further as you do?

Reply:
True. I didn't notice I throw error in user.ts.

Reply:
Based on your implementation in user.ts it will never return a typeof string.

Reply:
So I removed that part in your code and in other parts.